packages/simulator/index.js

```diff
@@ -37,35 +40,32 @@ const engine = {
     }
     statistics.collectExperimentMetadata(parameters)
 
-    const experiment = Object.assign(
-      {
-        virtualTime, // TODO: move this from being a static property to being a property of the experiment
-        cars: kommuner.pipe(mergeMap((kommun) => kommun.cars)),
-        dispatchedBookings: kommuner.pipe(
-          mergeMap((k) => k.dispatchedBookings)
-        ),
-        buses: regions.pipe(mergeMap((region) => region.buses)),
-        postombud,
-        kommuner,
-        parameters,
-        passengers: regions.pipe(mergeMap((region) => region.passengers)),
-        taxis: regions.pipe(mergeMap((region) => region.taxis)),
-      },
-      static
-    )
+    const experiment = {
+      virtualTime, // TODO: move this from being a static property to being a property of the experiment
+      cars: kommuner.pipe(mergeMap((kommun) => kommun.cars)),
+      dispatchedBookings: merge(
+        regions.pipe(mergeMap((r) => r.dispatchedBookings)),
+        kommuner.pipe(mergeMap((k) => k.dispatchedBookings))
+      ),
+      buses: regions.pipe(mergeMap((region) => region.buses)),
+      busStops: regions.pipe(mergeMap((region) => region.stops)),
```

Comment:
BusStops & lineShapes are part of static as defined above, part of faster restart feature. Please revert this change.